tests/test_app.py

```diff
@@ -0,0 +1,66 @@
+# tests/test_app.py
+
+import unittest
+import os
+from app import app
+
+os.environ['TESTING'] = 'true'
+
+
+class AppTestCase(unittest.TestCase):
+    def setUp(self):
+        self.client = app.test_client()
+
+    def test_home(self):
+        response = self.client.get("/")
+        assert response.status_code == 200
+        html = response.get_data(as_text=True)
+        assert "<title>Nicolas Ruiz</title>" in html
+        assert '<img width="125%" height="125%" src="./static/img/portrait.svg">'
+
+    def test_timeline(self):
+        response = self.client.get("/api/timeline_post")
+        assert response.status_code == 200
+        assert response.is_json
+        json = response.get_json()
+        assert "timeline_posts" in json
+        assert len(json["timeline_posts"]) == 0
+        response = self.client.post("/api/timeline_post",
+                                    data={"name": "John", "email": "john@example.com",
+                                          "content": "Hello world, I'm John!"})
+        assert response.status_code == 200
+        response = self.client.get("/api/timeline_post")
+        assert response.is_json
+        json = response.get_json()
+        assert "timeline_posts" in json
+        assert len(json["timeline_posts"]) == 1
+        assert json["timeline_posts"][0]["name"] == "John"
+        # TODO Add more tests relating to the /api/timeline_post GET and POST apis
+        # TODO Add more tests relating to the timeline page
+
+    def test_malformed_timeline_post(self):
+        # POST request missing name
+        response = self.client.post("/api/timeline_post",
+                                    data={"email": "john@example.com", "content": "Hello world, I'm John!"})
+        assert response.status_code == 400
+        html = response.get_data(as_text=True)
+        assert "Invalid name" in html
```

Comment:
The malformed post test could've passed easily if you had looked at my code and just replaced `invalid name` and `invalid content` with `Empty field`. I'm glad the invalid email one worked, though.👍